spec/spec_helper.rb

```diff
@@ -1,3 +1,11 @@
+# run coverage when on CI
+if ENV['CI']
+  require 'simplecov'
+  SimpleCov.start 'rails' do
+    add_filter '/spec/'
```

Comment:
Should this also run over `vendor/engines/**/spec/`? Or does this filter already hit it?